src/qt_gui/translations/pl_PL.ts

```diff
@@ -16,322 +16,322 @@
-            <translation>This software should not be used to play games you have not legally obtained.</translation>
+            <translation>Nie powinien być używany w celu ogrywania nielegalnych kopii gier.</translation>
         </message>
     </context>
     <context>
         <name>ElfViewer</name>
         <message>
             <location filename="../elf_viewer.cpp" line="45"/>
             <source>Open Folder</source>
-            <translation>Open Folder</translation>
+            <translation>Otwórz folder</translation>
         </message>
     </context>
     <context>
         <name>GameInfoClass</name>
         <message>
             <location filename="../game_info.cpp" line="26"/>
             <source>Loading game list, please wait :3</source>
-            <translation>Loading game list, please wait :3</translation>
+            <translation>Ładowanie listy gier. Prosimy cierpliwie czekać :3</translation>
         </message>
         <message>
             <location filename="../game_info.cpp" line="26"/>
             <source>Cancel</source>
-            <translation>Cancel</translation>
+            <translation>Wyjdź</translation>
         </message>
         <message>
             <location filename="../game_info.cpp" line="27"/>
             <source>Loading...</source>
-            <translation>Loading...</translation>
+            <translation>Ładowanie...</translation>
         </message>
     </context>
     <context>
         <name>GameInstallDialog</name>
         <message>
             <location filename="../game_install_dialog.cpp" line="24"/>
             <source>shadPS4 - Choose directory</source>
-            <translation>shadPS4 - Choose directory</translation>
+            <translation>shadPS4 - Wybierz ścieżkę</translation>
         </message>
         <message>
             <location filename="../game_install_dialog.cpp" line="31"/>
             <source>Directory to install games</source>
-            <translation>Directory to install games</translation>
+            <translation>Miejsce instalacji gier</translation>
         </message>
         <message>
             <location filename="../game_install_dialog.cpp" line="50"/>
             <source>Browse</source>
-            <translation>Browse</translation>
+            <translation>Przeglądaj</translation>
         </message>
         <message>
             <location filename="../game_install_dialog.cpp" line="74"/>
             <source>Error</source>
-            <translation>Error</translation>
+            <translation>Błąd</translation>
         </message>
         <message>
             <location filename="../game_install_dialog.cpp" line="75"/>
             <source>The value for location to install games is not valid.</source>
-            <translation>The value for location to install games is not valid.</translation>
+            <translation>Ścieżka jest nieprawidłowa.</translation>
```

Comment:
I think the source states something about the path and that information is omitted in translation.

Reply:
I agree that there is a difference to the original, though the english source doesn't sound right and chosen translation might be the improvement overall.  Probably original should be  "Location of games directory is invalid." or  "Path to games location folder is invalid." etc.